Src/Tests/DeUrgenta.RecurringJobs.Tests/Jobs/EventArchivalJobShould.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DeUrgenta.Domain;
+using DeUrgenta.Domain.Entities;
+using DeUrgenta.RecurringJobs.Domain;
+using DeUrgenta.RecurringJobs.Domain.Entities;
+using DeUrgenta.RecurringJobs.Jobs;
+using DeUrgenta.RecurringJobs.Jobs.Config;
+using DeUrgenta.RecurringJobs.Tests.Builders;
+using DeUrgenta.Tests.Helpers;
+using DeUrgenta.Tests.Helpers.Builders;
+using FluentAssertions;
+using Microsoft.Extensions.Options;
+using NSubstitute;
+using Xunit;
+
+namespace DeUrgenta.RecurringJobs.Tests.Jobs
+{
+    [Collection(TestsConstants.DbCollectionName)]
+    public class EventArchivalJobShould
+    {
+        private readonly DeUrgentaContext _context;
+        private readonly JobsContext _jobsContext;
+
+        public EventArchivalJobShould(JobsDatabaseFixture fixture)
+        {
+            _context = fixture.Context;
+            _jobsContext = fixture.JobsContext;
+        }
+
+        [Fact]
+        public async Task Archive_event_when_the_event_is_due()
+        {
+            //Arrange
+            var eventType = new EventType {Name = "TestType"};
+            eventType = _context.EventTypes.AddAsync(eventType).Result.Entity;
+            await _context.SaveChangesAsync();
+            
+            var eventEntity = new Event
```

Comment:
Would suggest creating an EventBuilder to make it more clear which fields are actually relevant for the test setup. Can use CertificationBuilder or UserBuilder as examples.